doc/en/developer/source/policies/gsip_template.txt

```diff
@@ -0,0 +1,47 @@
+# GSIP 1234 - Title
```

Comment:
We already have a template in the wiki, and the proposals page links to it:
https://github.com/geoserver/geoserver/wiki/GSIP-XXXX

Why duplicate it?

Reply:
Good to remove it from the wiki, as you can not cut and paste right from there. Only reason this template change is when the list of module maintainers changes.

Reply:
Good to remove it from the wiki, as you can not cut and paste right from there. Only reason this template change is when the list of module maintainers changes.